ci-operator/step-registry/upi/install/azurestack/upi-install-azurestack-commands.sh

```diff
@@ -0,0 +1,283 @@
+    --suffix-storage-endpoint "ppe3.stackpoc.com" 
+az cloud set -n PPE
+az cloud update --profile 2019-03-01-hybrid
+az login --service-principal -u $APP_ID -p $AAD_CLIENT_SECRET --tenant $TENANT_ID > /dev/null
+
+# shellcheck disable=SC2034
+SUBSCRIPTION_ID=$(az account show | jq -r .id)
+export SUBSCRIPTION_ID
+export AZURE_AUTH_LOCATION="${SHARED_DIR}/osServicePrincipal.json"
+
+echo "Installing python modules: yaml"
+python3 -c "import yaml" || pip3 install --user pyyaml
+
+# remove workers from the install config so the mco won't try to create them
+python3 -c '
+import yaml;
+path = "install-config.yaml";
+data = yaml.full_load(open(path));
+data["compute"][0]["replicas"] = 0;
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+openshift-install create manifests
+
+# we don't want to create any machine* objects 
+rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+
+python3 -c '
+import sys, json, yaml, os;
+path = "manifests/cloud-provider-config.yaml";
+data = yaml.full_load(open(path));
+config = json.loads(data["data"]["config"]);
+config["cloud"] = "AzureStackCloud";
+config["tenantId"] = os.environ["TENANT_ID"];
+config["subscriptionId"] = os.environ["SUBSCRIPTION_ID"];
+config["location"] = os.environ["AZURE_REGION"];
+config["aadClientId"] = os.environ["AAD_CLIENT_ID"];
+config["aadClientSecret"] = os.environ["AAD_CLIENT_SECRET"];
+config["useManagedIdentityExtension"] = False;
+config["useInstanceMetadata"] = False;
+config["loadBalancerSku"] = "basic";
+data["data"]["config"] = json.dumps(config);
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+python3 -c '
+import yaml,os;
+path = "manifests/cluster-config.yaml";
+data = yaml.full_load(open(path));
+install_config = yaml.full_load(data["data"]["install-config"]);
+platform = install_config["platform"];
+platform["azure"]["cloudName"] = "AzureStackCloud";
+platform["azure"]["region"] = os.environ["AZURE_REGION"];
+install_config["platform"] = platform;
+data["data"]["install-config"] = yaml.dump(install_config);
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+
+python3 -c '
+import base64, yaml,os;
+path = "openshift/99_cloud-creds-secret.yaml";
+data = yaml.full_load(open(path));
+data["data"]["azure_subscription_id"] = base64.b64encode(os.environ["SUBSCRIPTION_ID"].encode("ascii")).decode("ascii");
+data["data"]["azure_client_id"] = base64.b64encode(os.environ["AAD_CLIENT_ID"].encode("ascii")).decode("ascii");
+data["data"]["azure_client_secret"] = base64.b64encode(os.environ["AAD_CLIENT_SECRET"].encode("ascii")).decode("ascii");
+data["data"]["azure_tenant_id"] = base64.b64encode(os.environ["TENANT_ID"].encode("ascii")).decode("ascii");
+data["data"]["azure_region"] = base64.b64encode(os.environ["AZURE_REGION"].encode("ascii")).decode("ascii");
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
```

Comment:
Remove this too, but we will also have to create secrets manually. 

The CCO/Hive team has decided that all new platforms will run in `Manual` Credentials Mode, which means users must:
1. extract credentials requests from the release image
2. after running `create manifests`, create a secret for each credentials request

This is described here: https://github.com/openshift/installer/blob/d5bb5cf68e00f3cafa6b7ced75e47d2a697a957e/docs/user/azure/install_upi_azurestack.md#create-cluster-credentials

and it has a link to the official docs.

For CI, we will need to write a bash script to do this dynamically. I can help with that.